ts/IterableInbox.tsx

```diff
@@ -208,6 +208,15 @@ const IterableInbox = ({
       setIsMessageDisplay(false)  
    }
 
+   function updateCurrentVisibleRows() {
+      // inboxDataModel.updateVisibleRows(getCurrentVisibleRows())
+   }
+
+   function getCurrentVisibleRows(): InboxImpressionRowInfo[] {
+
+      return []
+   }
+
```

Comment:
This is for the implementation for the silent push correct?

Reply:
No, it's part of the upcoming session tracking